docs/pages/versions/unversioned/sdk/cellular.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Cellular
+---
+
+Provides information about the user’s cellular service provider, such as its unique identifier, cellular connection type and whether it allows VoIP calls on its network.
+
+Note: On iOS simulator, cellular network information cannot be retrieved. 
+
+## Installation
+
+This API is pre-installed in [managed](../../introduction/managed-vs-bare/#managed-workflow) apps. To use it in a [bare](../../introduction/managed-vs-bare/#bare-workflow) React Native app, follow its [installation instructions](https://github.com/expo/expo/tree/master/packages/expo-device).
+
+## API
+
+```js
+import * as Cellular from 'expo-cellular';
+```
+
+### Constants
+
+- [`Cellular.allowsVoip`](#cellularallowsvoip)
+- [`Cellular.carrier`](#cellularcarrier)
+- [`Cellular.isoCountryCode`](#cellularisocountrycode)
+- [`Cellular.mobileCountryCode`](#cellularmobilecountrycode)
+- [`Cellular.mobileNetworkCode`](#cellularmobilenetworkcode)
+
+### Methods
+
+- [`Cellular.getCellularGenerationAsync()`](#cellulargetcellulargenerationasync)
+
+### Enum Types
+
+- [`Cellular.CellularGeneration`](#cellularcellulargeneration)
+
+### Errors
+
+- [Error Codes](#error-codes)
+
+## Constants
+
+### `Cellular.allowsVoip`
+
+Indicates if the carrier allows making VoIP calls on its network. On Android, this checks whether the system supports SIP-based VoIP API. See [here](https://developer.android.com/reference/android/net/sip/SipManager.html#isVoipSupported(android.content.Context)) to view more information. 
+
+On iOS, if you configure a device for a carrier and then remove the SIM card, this property retains the `boolean` value indicating the carrier’s policy regarding VoIP. If you then install a new SIM card, its VoIP policy `boolean` replaces the previous value of this property.
+
+On web, this returns `null`.
+
+#### Examples
+
+```js
+Cellular.allowsVoip; // true or false
+```
+
+### `Cellular.carrier`
+
+The name of the user’s home cellular service provider. If the device has dual sim card, only the current active sim card on the network will be returned. On Android, this value is only available when SIM state is [`SIM_STATE_READY`](https://developer.android.com/reference/android/telephony/TelephonyManager.html#SIM_STATE_READY). Otherwise, this returns `null`. 
```

Comment:
```suggestion
The name of the user’s home cellular service provider. If the device has dual SIM cards, only the carrier for the currently active SIM card will be returned. On Android, this value is only available when the SIM state is [`SIM_STATE_READY`](https://developer.android.com/reference/android/telephony/TelephonyManager.html#SIM_STATE_READY). Otherwise, this returns `null`. 
```